integration/docker/Dockerfile

```diff
@@ -57,7 +52,7 @@ ARG ALLUXIO_UID=1000
 ARG ALLUXIO_GID=1000
 ARG ENABLE_DYNAMIC_USER=true
 
-# Add Tini for Alluxio helm charts
+# add Tini for Alluxio helm charts (https://github.com/Alluxio/alluxio/pull/12233)
```

Comment:
Add ?

Reply:
The same issue as before, if one uses lowercase, all use lowercase.
If one uses uppercase, all use uppercase, figure out the code convention